save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/TopBar.kt

```diff
@@ -54,6 +54,148 @@ external interface TopBarProps : PropsWithChildren {
     var userInfo: UserInfo?
 }
 
+/**
+ * The class for analyzing url address and creating links in topBar
+ */
+class TopBarUrl(href: String) {
+    /**
+     * currentPath is the link that we put in buttons
+     */
+    var currentPath = "#"
```

Comment:
you use `var` in frontend as modifying variable. I'm not sure it will work, we have to use `state` for storing content

Reply:
It works

Reply:
In case of emergency you will be the one who fixes it so its ok if you are sure it works

Reply:
good